dask/highlevelgraph.py

```diff
@@ -527,7 +537,8 @@ def layer_info_dict(self):
                 info[key] = html.escape(str(val))
         if self.collection_annotations is not None:
             for key, val in self.collection_annotations.items():
-                info[key] = html.escape(str(val))
+                if key != "chunks":
```

Comment:
Please add a comment explaining why we do this

Reply:
Also, this isn't super extendable if we add other keys to the dictionary that we also want to skip. If we did that, you might want a list of key names to skip, and then check `if key is in keys_to_skip: ...`
Not sure if we need to do that here (probably not, but you need to understand how that would happen if we needed it).

Reply:
1. Added a comment. 
2. yes, you are right. It would be a lot cleaner if I had a list of keys to not include in the HTML. If we end up having more than 1, i will use a list instead as suggested :heavy_check_mark: